config.yaml

```diff
@@ -0,0 +1,18 @@
+
+
+OPENAI_API_KEY=""
```

Comment:
Can you add the usage of this config alongside it in the same PR?

Reply:
Alright. will do.